tick/optim/model/src/hawkes_fixed_expkern_leastsq.h

```diff
@@ -76,8 +76,8 @@ class ModelHawkesFixedExpKernLeastSq : public ModelHawkesSingle {
   void grad_i(const ulong i, const ArrayDouble &coeffs, ArrayDouble &out) override;
```

Comment:
Hum, I think it computes grad_i, documentation is for the function that comes after void hessian(ArrayDouble &out)